### ISSUE TITLE
unknown column names in an SBE CTD file

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1948]
I saw the following, in reading a test file that was provided to me for issue #1947.  I'll look in the docs to see if the guess is correct, and I'll try to find what the other thing means.
```
1: In cnvName2oceName(lines[nameLines[iline]], columns, debug = debug -  :
  assuming 'sbox0Mm/Kg' is equivalent to 'sbeox0Mm/Kg'
2: In cnvName2oceName(lines[nameLines[iline]], columns, debug = debug -  :
  unrecognized SBE name 'sbox0dV/dT'; consider using 'columns' to define this na
me   
```

**Comment by dankelley** [comment]
I do not see either of
* `sbox0dV/dT`
* `sbox0Mm/Kg`

in the Seasoft manual I have available (attached).  (I did a search on `sbox` and it turned up noting.)

However, that manual is quite old, dated 2017, and perhaps there are updates.

I wonder whether @richardsc or @akrenner has a more recent version of the SBE docs (in PDF) they can send me. 

Some notes:

1. These items are seen in data files @akrenner sent for issue #1947. My policy is to try to clear up such issues whenever they arise.  I do not typically try to insert all possible parameter names, because there are a lot of them (see the 12 pages of Appendix VI in the attached), and life is short.

2. I am putting the two test files that @akrenner sent into the `local_data` directory, so I can perform build-time tests on it.  In particular, I am testing that the new code makes a warning for those oddly-formated dates.  But it also warns about not understanding the above-named quantities.  If I cannot make oce understand them, I will need to add the warnings to the test suite.

3. None of this work is pushed to github yet.  I will push it later today, though.  I need to do more testing locally.  (I have a thousand or so ctd files on my machine. None had that odd time format we discovered in #1947, though. That's why I'm grateful to @akrenner for sending the files.  As an aside, the new code will not only detect impossible-to-decode times, but also impossible-to-have times, e.g. a time earlier than 1900 (which can happen easily, since e.g. "01-02-2022" translates in R/`as.POSIXct()` to year 1, month 2, day 20).

[SBEDataProcessing_7.26.7.pdf](https://github.com/dankelley/oce/files/8747688/SBEDataProcessing_7.26.7.pdf)
